coalib/collecting/Collectors.py

```diff
@@ -452,7 +452,10 @@ def collect_registered_bears_dirs(entrypoint):
     for ep in pkg_resources.iter_entry_points(entrypoint):
         registered_package = None
         try:
-            registered_package = ep.load()
+            # backwards compat with old setuptools
+            # (that do not provide Entrypoint.resolve)
```

Comment:
Remove # comments as you must have seen that there are no comments present in coala code file.

Reply:
... that's *on purpose*?

Reply:
I'm confused as comments are generally in the entire tech industry considered to be a *good thing*. Did I understand you correctly?

Reply:
Also, there are actually already comments in this file. Check L27+40

Reply:
@Dhiraj240 The comments are a necessary part of the commit. And as @akatrevorjay pointed out, coala code is generally filled with a lot of comments. Sorry for this @Dhiraj240 ,but this is not a proper review.
@akatrevorjay Please keep the comments intact. And thanks for adding comments to your code, it makes the review easier.

Reply:
@Dhiraj240  @Naveenaidu Probably it was my mistake to not to give a elaborative review, i meant to say that please remove comments written in ```#```  format and instead define your comments in the function itself like somewhere in between ``` """
     Searches setuptools for the entrypoint and returns the bear
     directories given by the module.
 
     :param entrypoint: The entrypoint to find packages with.
     :return:           List of bear directories.
     """
```

Reply:
@akatrevorjay @Naveenaidu Probably it was my mistake to not to give a elaborative review, i meant to say that please remove comments written in # format i.e single line comment instead use it before try as the comments in coala code files are written before any loop or scope.@Naveenaidu if you feel that it is not good then i am sorry as i just gave my point because i never follow single line comment in between declarations or functions or loops or statements.
Probably i have learnt a lesson to give elaborative reviews.

Reply:
@Dhiraj240 Thanks so much for clearing that up! Sorry about that. I can do that if you prefer, but the reason why I didn't is that it serves no use to the user of the implementation, ie it's an internal only "concern" that as a downstream API consumer you wouldn't benefit from knowing.

Reply:
Alright, fine.